MatrixPilot/MAVLink.c

```diff
@@ -650,7 +650,7 @@ static void handleMessage(void)
 	handling_of_message_completed |= MAVParamsHandleMessage(handle_msg);
 	handling_of_message_completed |= MAVMissionHandleMessage(handle_msg);
 	handling_of_message_completed |= MAVFlexiFunctionsHandleMessage(handle_msg);
-	handling_of_message_completed |= MAVFTPHandleMessage(handle_msg);
+//	handling_of_message_completed |= MAVFTPHandleMessage(handle_msg); // WIP - RobD

```

Comment:
Agreed. And fully agree with the philosophy TomP put forward. In general with code that needs to be deemed airworthy, 'dead code' is quite a no no. However in this particular instance, I'd like to keep this here as more a comment alluding to the fact that such functionality is on the table as work in progress, and as I believe it adds a little value as a place holder for where such would be called in.